src/index.js

```diff
@@ -0,0 +1,237 @@
+'use strict';
+
+const state = {
+  tempValue: 75,
+  color: 'orange',
+  landscape: '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷',
+  skyscape: '',
+  background: "url('../assets/summer.jpg')",
+  city: 'Orlando',
+  lat: '',
+  lon: '',
```

Comment:
You could also hard code the lat/lon for Orlando here instead of setting them to empty strings